example/docker-compose.yml

```diff
@@ -84,12 +84,22 @@ services:
       - IDENTITY_KEY=71aec985a12cfcbabc7969845215ceac56038e87477c8337e373eab7b15796cf
       - IDENTITY_KEY_TYPE=secp256k1
 
+#
+#  otel-collector:
```

Comment:
I would prefer if we use still otel collector to export to jaeger so we test that it works.

Reply:
yeah, added collector and also Prometheus to locally play with metrics.